vm-setup/roles/v1aX_integration_test/filter_plugins/status.py

```diff
@@ -0,0 +1,70 @@
+""" filtering k8s_info resources """
+
+from ansible.errors import AnsibleFilterError
+from ansible.utils.display import Display
+
+display = Display()
+
+
+def msg(key, resources):
+    return "could not find %r key in %r" % (key, resources)
+
+
+def filter_phase(resources, phase, skip_missing=True):
+    """return resources in a defined phase"""
+    filtered = []
+    for r in resources:
+        try:
+            if r["status"]["phase"].lower() == phase:
+                filtered.append(r)
+        except KeyError:
+            if skip_missing:
+                display.warning(msg("['status']['phase']", resources))
+                break
+            raise AnsibleFilterError(msg("['status']['phase']", resources))
```

Comment:
None of the code in this change runs with `skip_missing` False. Rather than write code for logic branches that aren't used at all (in the assumption that someone later will use your parameter), I'd rather we remove the `skip_missing` parameter entirely, and make the `skip_missing` behaviour the default and only behaviour, rather than giving users the option to have the filter plugin error.

Reply:
Next commit `Skip_missing` will be removed only keep `display.warning`